L_advanced_deployment.md

```diff
@@ -55,67 +56,65 @@ Now we need to update our `mix.exs` file to have all production dependencies lis
 ```elixir
   def application do
     [mod: {HelloPhoenix, []},
-      applications: [:phoenix, :cowboy, :logger, :postgrex, :phoenix_ecto, :phoenix_html]]
+     applications: [:phoenix, :cowboy, :logger, :postgrex, :phoenix_ecto, :phoenix_html]]
```

Comment:
The "mod:" to "applications:" indentation is the same as it is in a freshly generated `mix.exs`. I'm about to break the list of applications as it appears in `mix.exs` as well.